examples/deferred_processing/automated_deferred_processing/branch-mover.json

```diff
@@ -0,0 +1,33 @@
+{
+  "pipeline": {
+    "name": "branch-mover"
+  },
+  "input": {
+    "cron": {
+        "name": "tick",
+        "spec": "@every 1m",
+        "overwrite": true
+    }
+  },
+  "transform": {
+      "cmd": ["sh" ],
+      "stdin": [
+          "echo '{\"pachd_address\": \"grpc://pachd:650\"}' | pachctl config set context default --overwrite",
+          "echo ${PACHYDERM_AUTH_TOKEN} | pachctl auth use-auth-token",
+          "pachctl create branch edges_dp@master --head dev"
+      ],
+      "image": "pachyderm/pachctl:1.11.0",
+      "secrets": [ {
+          "name": "pachyderm-user-secret",
+          "mount_path": "/pachyderm-user-secret"
+      },
+      {
+          "name": "pachyderm-user-secret",
+          "env_var": "PACHYDERM_AUTH_TOKEN",
+          "key": "auth_token"
+      } ]
+  }
+
+}
```

Comment:
I'm not sure, but would it be possible to combine these two branch mover specs? i.e. mount the secret only if it exists?

Reply:
I don't think there are conditionals in a pipeline's use of secrets, it's all or nothing.

Reply:
Right, but what happens when you mount an env var that doesn't exist? If it just ignores it and the pipeline doesn't fail, then you could use one pipeline spec.